pymc3/__init__.py

```diff
@@ -34,3 +34,21 @@
     _log.setLevel(logging.INFO)
     handler = logging.StreamHandler()
     _log.addHandler(handler)
+
+def tqdm(*args, **kwargs):
+    try:
+        from tqdm import tqdm_notebook
+        return tqdm_notebook(*args, **kwargs)
+    except:
```

Comment:
Do you think this exception could be different on different systems?  I'm on Ubuntu fyi.